src/components/vote/VoteItem.js

```diff
@@ -1,35 +1,57 @@
+import * as firebase from 'firebase/app';
 import React, { Component } from 'react';
 import { Button, Card, Image } from 'semantic-ui-react';
 import DropdownExampleSelection from './Dropdown';
 import awardIcon from '../../images/grammys.png';
+import 'firebase/firestore';
+import './Vote.css';
 
 
-/* eslint-disable */
 class VoteItem extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { voters: [] };
+    this.db = firebase.firestore();
+  }
+
+  componentDidMount() {
+    const db = firebase.firestore();
```

Comment:
firestore has already been initialized here. You should be able to use `this.db` instead

Reply:
got it!